platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/editor/StateEditorResize.java

```diff
@@ -55,8 +60,14 @@ public void run() {
                     + parent.getClass().getSimpleName() + " and is not an internal node.");
         }
 
-        System.out.println("Resizing [" + MerkleRouteUtils.merkleRouteToPathFormat(parent.getRoute()) + " "
-                + parent.getClass().getSimpleName() + "] to child count " + newSize);
+        if (logger.isInfoEnabled(LogMarker.CLI.getMarker())) {
+            logger.info(
+                    LogMarker.CLI.getMarker(),
+                    "Resizing [{}{}] to child count {}",
```

Comment:
Previous log had a space between route and name `[{} {}]`

Reply:
Thanks :)